src/main/resources/spy.properties

```diff
@@ -0,0 +1,2 @@
+modulelist=com.p6spy.engine.spy.P6SpyFactory
+useNanoTime=true
```

Comment:
I'm trying to understand what this does. Can you explain a bit?

Reply:
P6Spy - monitors db changes. I believe it's so that we produce sentry db queries entries in the Performance view.
nanotime scale would be meaningful because https://github.com/agorapulse/grails-sentry/pull/56#discussion_r865816012

Reply:
It's actually for disabling p6spy logging to file. I will add comment there.

Reply:
b2c85e44